### ISSUE TITLE
[Bug]: ND_PLAYLISTSPATH: "../playlist/" does not work

### ISSUE THREAD

**Issue by tsuyo-o** [navidrome/navidrome#3195]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [X] I'm using the latest version (your issue may have been fixed already)

### Version

0.52.5 

### Current Behavior

I'm using docker.  When ND_PLAYLISTSPATH is "../playlist",  navidrome doesn't load m3u files under the directory.
Directory structure is like this.  "/music" is music directory and "/playlist" is playlist directory. So I expect navidrome recognizes "../playlist" as "/playlist".

### Expected Behavior

navidrome loads m3u files.

### Environment

```markdown
- OS: Docker (deluan/navidrome:latest)
- Browser:
- Client:
```

### How Navidrome is installed?

Docker

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by kgarner7** [comment]
Navidrome will only scan your music library (`MusicFolder`). `PlaylistsPath` needs to be relative to `MusicFolder` (as a child): https://www.navidrome.org/docs/usage/configuration-options/#:~:text=--,playlistspath,-ND_PLAYLISTSPATH

**Comment by tsuyo-o** [comment]
Thank you for the reply. I understand the specification. But the document only mentions about the relative path, not about the child path. This made me confused.

**Comment by kgarner7** [comment]
Yeah, that's fair. If you have suggestions for better documentation (or want to raise it further), consider filing an issue on https://github.com/navidrome/website
